### DISCUSSION TITLE
Recurring image policy notifications and event rate limiting

### DISCUSSION THREAD

**Discussion by ArtunSubasi** [fluxcd/flux2#1636] [Q&A]
Hi there,

I'm using Flux2 including the image automation controllers and Slack notifications. I configured image policies that check an Amazon Elastic Container Registry to detect the newest images. They work like a charm.

There is only one thing that is pretty annoying. The notification controller sends an Image Policy notification every 5 minutes even if there is no update on images. E.g. "the newest image is v1.1.0". Getting a similar notification 288 times a day pro image is a no-go for me. I had to suspend the events.

My expectation would be receiving an event if a new image is detected. Otherwise I get informed about news that is irrelevant which causes the team the ignore notifications pretty fast.

My questions:

* Is this a bug? If not, is it possible to make this configurable?
* I read about rate limiting [in the docs here](https://fluxcd.io/docs/components/notification/event/#rate-limiting). But I don't really get how to configure it. Where can I set the mentioned `--rate-limit-interval` option?

**Comment by alexclifford** [comment]
Did you manage to solve this issue @ArtunSubasi?

**Comment by ArtunSubasi** [comment]
Sadly no. I couldn't find solution and did not get a response. I ended up suspending the image policy notifications completely because the rate of notifications was pretty annoying and not helpful.

**Reply by stefanprodan** [reply]
Yes this has been reported here https://github.com/fluxcd/image-reflector-controller/issues/166 The workaround for now is to add only the ImageUpdateAutomation to the alert.
